package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "node-red-contrib-rx-nostr",
+  "name": "@hzrd149/node-red-contrib-rx-nostr",
```

I don't think I can publish the package under `@hzrd149` on npm
```suggestion
  "name": "node-red-contrib-rx-nostr",
```